sdk/file_sdk/Common/tag.cpp

```diff
@@ -12,26 +13,6 @@ static const string kExtendedPrefix = "Extended";
 

```

Comment:
move to mip namespace?

Reply:
Move what? CodeFlow doesn't show any line related to that comment

---
In reply to: [136713104](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136713104) [](ancestors = 136713104)

Reply:
kMetaDataPrefix and kExtendedPrefix

---
In reply to: [136713223](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136713223) [](ancestors = 136713223,136713104)

Reply:
Munjal moved it out

---
In reply to: [136934853](https://github.com/AzureAD/rms-sdk-for-cpp/pull/140#discussion_r136934853) [](ancestors = 136934853,136713223,136713104)